modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-template/en-US.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+
+<html lang="en">
+	<body>
+		<main>
+			<p>A Marketing Development Found request has been [re]submitted.</p>
+
+			<p>Please review the request at the following link:</p>
+
+			<p>
+			<a href="http://localhost:8080/web/prm/l/[%MDFCLAIM_ID%]">http://localhost:8080/web/prm/l/[%MDFCLAIM_ID%]</a>
+			</p>
```

Comment:
```suggestion
			<a href="http://localhost:8080/web/prm/l/[%MDFCLAIM_ID%]">http://localhost:8080/web/prm/l/[%MDFCLAIM_ID%]</a>
```

This HTML is not good. A p tag with a tag. Did you test this?
Probably it best to remove this p tag.